From 76286e78c6e5b1ab663707a0088c7b20fdd84127 Mon Sep 17 00:00:00 2001
From: Claudiu Manoil <claudiu.manoil@nxp.com>
Date: Fri, 4 Oct 2019 16:22:26 +0300
Subject: [PATCH 227/245] felix: Remove unused include, fix compile warn
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

commit 83800e10bb1c30fb2310cc20655966821f57ccc4 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux.git

Remnant from initial felix code that used fixed links for phys.
Fixes:
include/linux/phy_fixed.h:26:17: warning: ‘struct gpio_desc’ declared inside
				 parameter list will not be visible outside
				 of this definition or declaration
				 struct gpio_desc *gpiod);

Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/net/ethernet/mscc/felix_board.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/ethernet/mscc/felix_board.c b/drivers/net/ethernet/mscc/felix_board.c
index cdea8fe61622..a3f9d473cd85 100644
--- a/drivers/net/ethernet/mscc/felix_board.c
+++ b/drivers/net/ethernet/mscc/felix_board.c
@@ -7,7 +7,6 @@
 #include <linux/module.h>
 #include <linux/pci.h>
 #include <linux/netdevice.h>
-#include <linux/phy_fixed.h>
 #include <linux/phy.h>
 #include <linux/of_mdio.h>
 #include <linux/of_net.h>
-- 
2.17.1

